### PR TITLE
Docs: fix Hyperlink jump error in docs/flink/flink-getting-started.md

### PR DESCRIPTION
**Pull request by hunter-cloud09**
- fix Hyperlink jump error in docs/flink/flink-getting-started.md #5714

### CODE REVIEW

File: docs/flink/flink-getting-started.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ url: flink
 # Flink
 
 Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API. Currently,
-Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg also support Flink 1.11.
+Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg
```
Comment (pvary) [inline docs/flink/flink-getting-started.md:26]: Maybe we should update the currently supported versions here

**Comment by pvary** [comment]
@hunter-cloud09: Why do we have changes in linebreaks? Makes the review really hard. If they are not necessary, please remove them.  OTOH, it would be good to update the supported Flink versions on this page.

**Comment by hunter-cloud09** [comment]
> @hunter-cloud09: Why do we have changes in linebreaks? Makes the review really hard. If they are not necessary, please remove them. OTOH, it would be good to update the supported Flink versions on this page.

I know that, bug when i save changes to the doc, it automatically format, I don't know if the project has added formatting tools. and will fix them laster.

File: docs/flink/flink-getting-started.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ url: flink
 # Flink
 
 Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStream API and Table API. Currently,
-Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, and 1.14. Previous versions of Iceberg also support Flink 1.11.
+Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13, 1.14, and 1.15. Previous versions of Iceberg also support Flink 1.11.
```
[the remark this answers is not in the document]
Comment (hunter-cloud09) [reply docs/flink/flink-getting-started.md:26]: yep

**Comment by pvary** [comment]
@hunter-cloud09: I see that we have a conflict?
Could you please rebase?
Otherwise this looks good to me.
Thanks, 
Peter

**Comment by hunter-cloud09** [comment]
hi, @pvary ,I just pulled the latest code and found that the doc has been changed, so I closed the pr ^ _ ^, but the jump on the official website is still incorrect

**Comment by hunter-cloud09** [comment]
hi, ### @pvary , I saw it's changed in master branch, but in other branch is not, so should i commit into other branch?

**Comment by hunter-cloud09** [comment]
so , i did this [pr](https://github.com/apache/iceberg/pull/5724)

**Pull request closed without merging**